extensions/amp-story/1.0/amp-story.css

```diff
@@ -881,13 +881,17 @@ amp-story .amp-video-eq,
   --reaction-chip-shadow: none !important;
   --reaction-answer-choice-border: var(--reaction-accent-color) !important;
   --reaction-answer-choice-background: white !important;
+  --reaction-main-color: #005AF0;
```

Comment:
Is there a reason for this to live in amp-story instead of amp-story-reaction-poll-binary?

Reply:
I assume we will want to abstract some behaviors such as dark themes and shadow styles, so that we can have a consistent color palette and alternative styles. Even setting custom colors for backgrounds or text could be abstracted away. That could all be well thought and introduced on a future PR, but for now I'll move it to the reaction.css file since it belongs there.

Reply:
Update post-development: The code was moved to amp-story-reaction but this needed a small refactor that allows both CSSs from the abstract class and concrete class to coexist and also moving all the flags (theme, chip-style) from the element to the shadow root. Added all of that.